### PR TITLE
remove extra node config layer

### PR DESCRIPTION
**Pull request by deads2k**
Removes an extra layer we no longer need from the start node path.

@sjenning fyi
/assign soltysh

### CODE REVIEW

File: pkg/cmd/server/kubernetes/node/node.go [hunk 1]
```diff
@@ -4,34 +4,29 @@ import (
...
+	"k8s.io/kubernetes/cmd/kubelet/app"
 	"k8s.io/kubernetes/pkg/volume"
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
-	dockerutil "github.com/openshift/origin/pkg/cmd/util/docker"
 	"github.com/openshift/origin/pkg/volume/emptydir"
 )
 
-const minimumDockerAPIVersionWithPullByID = "1.22"
```
Comment (mfojtik) [inline pkg/cmd/server/kubernetes/node/node.go]: we don't check for the docker version anymore? or is this part of upstream

Comment (deads2k) [reply pkg/cmd/server/kubernetes/node/node.go]: > we don't check for the docker version anymore? or is this part of upstream

If we need it, we'll need need to move it upstream.  I'm 95% sure that our RPMS and ansible ensure the prereq level already.  @sdodson

Comment (soltysh) [reply pkg/cmd/server/kubernetes/node/node.go]: I'd be surprised if upstream didn't check that as well.

Comment (soltysh) [reply pkg/cmd/server/kubernetes/node/node.go]: And here you have it: https://github.com/openshift/origin/blob/8b81ebdcc7c1c70c05f12fd4e47a7d85ad9117fe/vendor/k8s.io/kubernetes/pkg/kubelet/cm/container_manager_linux.go#L708

File: pkg/cmd/server/kubernetes/node/node.go [hunk 2]
```diff
@@ -4,34 +4,29 @@ import (
...
-
+// TODO this is a best effort check at the moment that should either move to kubelet or be removed entirely
 // EnsureKubeletAccess performs a number of test operations that the Kubelet requires to properly function.
 // All errors here are fatal.
-func (c *NodeConfig) EnsureKubeletAccess() {
-	if c.Containerized {
+func EnsureKubeletAccess() {
+	containerized := cmdutil.Env("OPENSHIFT_CONTAINERIZED", "") == "true"
+	if containerized {
```
Comment (soltysh) [inline pkg/cmd/server/kubernetes/node/node.go]: Can you just squash this check here?

**Pull request merged**